routes.js

```diff
@@ -45,12 +46,13 @@ router.get("/inbox", RequireAuth, async (req, res) => {
         userId: "me",
         maxResults: 10,
         labelIds: "INBOX",
+        q:q,
         includeSpamTrash: false
     });
 
     // Iterate through the messages and pull down more information
     const inbox = [];
-    const messages = messageResponse.data.messages;
+    const messages = messageResponse.data.messages || [];
```

Comment:
If you were to type in something that did not return a result, the messages object from `messageResponse.data` could be null. To avoid this issue, we need to fall back to an empty array.